### PR TITLE
update changelog for M27 release

### PR DESCRIPTION
**Pull request by chliangGoogle**

### CODE REVIEW

File: Firebase/Messaging/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2015-06-12 -- v3.0.2
+- Add support for global data collection switch in FIRApp.
```
Comment (morganchen12) [inline Firebase/Messaging/CHANGELOG.md]: Consider rephrasing to 
```
Added support for enabling or disabling global data collection in `FIRApp`.
```

File: Firebase/Messaging/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Adding warning message when topic format is not right.
```
Comment (morganchen12) [inline Firebase/Messaging/CHANGELOG.md]: Consider rephrasing to `Added a warning message when subscribing to topics with incorrect name formats.`

File: Firebase/Messaging/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+- Adding deprecation warning to suppress deprecated token method usage inside FIRMessaging.
```
Comment (morganchen12) [inline Firebase/Messaging/CHANGELOG.md]: Consider rephrasing to `Silenced a deprecation warning in FIRMessaging.`

File: Firebase/Messaging/CHANGELOG.md (hunk 2 above)
Comment (chliangGoogle) [reply Firebase/Messaging/CHANGELOG.md]: Done

File: Firebase/Messaging/CHANGELOG.md (hunk 3 above)
Comment (chliangGoogle) [reply Firebase/Messaging/CHANGELOG.md]: Done.

**Pull request merged**